playbooks/common/openshift-master/config.yml

```diff
@@ -98,6 +111,24 @@
     set_fact:
       l_etcd3_enabled: "{{ etcd3_grep.rc == 0 | bool }}"
 
+  - name: Preserve OPENSHIFT_DEFAULT_REGISTRY master parameter if present
+    command: awk '/^OPENSHIFT_DEFAULT_REGISTRY/' /etc/sysconfig/atomic-openshift-master
+    register: l_default_registry_defined
+
+  - name: Preserve OPENSHIFT_DEFAULT_REGISTRY master-api parameter if present
+    command: awk '/^OPENSHIFT_DEFAULT_REGISTRY/' /etc/sysconfig/atomic-openshift-master-api
+    register: l_default_registry_defined_api
+
+  - name: Preserve OPENSHIFT_DEFAULT_REGISTRY master-controllers parameter if present
+    command: awk '/^OPENSHIFT_DEFAULT_REGISTRY/' /etc/sysconfig/atomic-openshift-master-controllers
+    register: l_default_registry_defined_controllers
+
+  - name: Update facts with OPENSHIFT_DEFAULT_REGISTRY value
+    set_fact:
+      l_default_registry_value: "{{ l_default_registry_defined.stdout }}"
+      l_default_registry_value_api: "{{ l_default_registry_defined_api.stdout }}"
+      l_default_registry_value_controllers: "{{ l_default_registry_defined_controllers.stdout }}"
+
```

Comment:
This whole section is to ensure

* if `OPENSHIFT_DEFAULT_REGISTRY=<default_value:here>` in your existing settings, then nothing really changes
* If there is NO VALUE OR REFERENCE TO `OPENSHIFT_DEFAULT_REGISTRY` AT ALL then no lines are added on the scaleup master
* If you have a value that is non-standard, that value is used instead of the default